include/zephyr/mgmt/mcumgr/grp/os_mgmt/os_mgmt_client.h

```diff
@@ -10,6 +10,13 @@
 #include <inttypes.h>
 #include <zephyr/mgmt/mcumgr/smp/smp_client.h>
 
+/**
+ * @brief MCUmgr os_mgmt_client API
```

Comment:
Nit: IMO it would be useful to actually use the brief for a more human-readable and meaningful text than the group "technical" name, i.e at a minimum spell things out.


```suggestion
 * @brief MCUmgr OS management client API
```


This also applies to all other groups I guess.